README.md

```diff
@@ -1,2 +1,62 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+PROGRAM Terminal
+
+Init Terminal
+
+Guesses = 4
+
+Correct Word = Marks
+
+Words = Starks, Stake, Helps, Empty, Marks
+
+Print “You have 4 guesses”
+
+Guess = Input Guess
+
+
+If chosen word has no characters right 
+guessesTaken = guessesTaken +1
+
+Then 
+Print “0/5 characters correct”
+
+If chosen word has 1 characters right 
```

Comment:
This approach assumes a word length of 5 letters, and would be difficult to scale to different lengths. It is worth trying to find a more concise implementation that works for any length of word.